src/ogma/features/nodeGrouping.ts

```diff
@@ -79,7 +85,9 @@ export class NodeGroupingTransformation {
             }
           };
         },
-        showContents: true,
+        showContents: (metaNode) => {
+          return OgmaTools.isGroupCollapsed(metaNode);
```

Comment:
```suggestion
          return !OgmaTools.isGroupCollapsed(metaNode);
```
not sure, I guess you tested it, so I might be wrong. But it feels you want to show the content of the group is the group is NOT collapsed

Reply:
you're right, I fixed it locally and forget to push